api/enki.py

```diff
@@ -101,6 +102,12 @@ class EnkiAPI(BaseCirculationAPI, HasSelfTests):
         (epub, adobe_drm): 'acs',
     }
 
+    # Enki API serves all responses with a 200 error code and a
+    # text/html Content-Type. However, there's a string that
+    # reliably shows up in error pages which is unlikely to show up
+    # in normal API operation.
+    ERROR_INDICATOR = '<h1>Oops, an error occurred</h1>'
+
```

Comment:
This is the core of the fix to https://jira.nypl.org/browse/SIMPLY-1848. I'm not super happy with it, but it's okay.

Reply:
Yeah, this sounds annoying to get all responses as 200.